src/Error/Error.php

```diff
@@ -124,16 +124,10 @@ private function updateRepr(): void
         }
 
         if ($this->source && $this->source->getName()) {
-            $name = $this->source->getName();
-            if (\is_string($name) || $name instanceof \Stringable) {
-                $name = \sprintf('"%s"', $name);
-            } else {
-                $name = json_encode($name);
-            }
-            $this->message .= \sprintf(' in %s', $name);
+            $this->message .= \sprintf(' in "%s"', $this->source->getName());
         }
 
-        if ($this->lineno && $this->lineno >= 0) {
```

Comment:
As we cannot have `!$this->lineno` and `$this->lineno = 0` at the same time

Reply:
Indeed, it's all because we've added type hints which makes such code irrelevant.